src/applications/personalization/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -19,23 +19,24 @@ class RatedDisabilityList extends React.Component {
   }
 
   noDisabilityRatingContent(errorCode) {
-    let headline;
     let status;
     let content;
     if (isServerError(errorCode)) {
-      headline = 'We’re sorry. Something went wrong on our end';
       status = 'error';
       content = (
         <>
+          <h2 className="vads-u-margin-y--0 vads-u-font-size--lg">
+            We’re sorry. Something went wrong on our end
+          </h2>
           <p>
             Please refresh this page or check back later. You can also sign out
             of VA.gov and try signing back into this page.
           </p>
           <p>
             If you get this error again, please call VA.gov help desk at{' '}
             <a
-              href="tel:1-855-574-7286"
-              aria-label="Dial the telephone number 1-855-574-7286"
+              href="tel:18555747286"
+              aria-label="1. 8 5 5. 5 7 4. 7 2 8 6."
```

Comment:
This is potentially a better way to approach accessible phone numbers - [more here](https://thatdevgirl.com/blog/accessibility-phone-number-formatting)

Reply:
@1Copenut - for visibility, regarding aria-label and phone numbers.

Reply:
👍 I just read the linked article and think this should be our pattern for all `tel` links.

Reply:
Thanks for finding the article @jenstrickland! Was super helpful!